src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color color = Color.values()[new Random().nextInt(Color.values().length)];
+        return color.toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum, you should use the `name()` method. The `name()` method is designed for this purpose and will always return the exact name of the enum constant as declared in the enum declaration.